packages/cli/package.json

```diff
@@ -40,8 +40,7 @@
     "node": ">=16"
   },
   "bin": {
-    "hs": "./bin/hs",
-    "hscms": "./bin/hscms"
```

Comment:
Do `CMS` devs not use this?  Maybe we could delete the `bin/hscms` file as well.  It seems to be identical to `bin/hs` except it doesn't have the `--no-deprecation` flag.

Reply:
Ok actually fully did not realize that running commands with `hscms` was something you could still do - thought this was left over from an older setup.

We probably _should_ remove that functionality, but it would be a whole thing and require a major version bump, so I'll leave it out of this PR